<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgPreferences</class>
 <widget class="QDialog" name="dlgPreferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="gboxFont">
     <property name="title">
      <string>Font</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="chkReaderFont">
        <property name="styleSheet">
         <string notr="true">QCheckBox::indicator {
    /* 选择框尺寸 */
    width:22px;
    height:22px;
}</string>
        </property>
        <property name="text">
         <string>Use reader font to override UI font</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblFontDemo">
        <property name="text">
         <string>Font Size</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sliderFontSize">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QSlider::groove:horizontal 
{ 
	border: 1px solid #999999; 
	height: 2px; 
	margin: 0px 0;
	left: 5px; 
	right: 5px; 
}  

QSlider::add-page:horizontal
{
	background-color: gray;
}

QSlider::sub-page:horizontal
{
	background-color: #1AFA29;
}

QSlider::handle:horizontal
{
	border: 0px ;
	border-image: url(:/res/qslider_btn.png);			
	width: 20px; 
	margin: -10px -5px -10px -5px; 
	
}</string>
        </property>
        <property name="minimum">
         <number>13</number>
        </property>
        <property name="maximum">
         <number>30</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>16</number>
        </property>
        <property name="tracking">
         <bool>false</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnCustomFont">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Custom Font</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblCustomFont">
        <property name="text">
         <string>Font Path</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblTip">
        <property name="text">
         <string>The font setting takes effect after reopening the app.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Additional Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QCheckBox" name="chkClose">
        <property name="styleSheet">
         <string notr="true">QCheckBox::indicator {
    /* 选择框尺寸 */
    width:22px;
    height:22px;
}</string>
        </property>
        <property name="text">
         <string>Normal Close</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="chkAutoTime">
        <property name="styleSheet">
         <string notr="true">QCheckBox::indicator {
    /* 选择框尺寸 */
    width:40px;
    height:40px;
}

QCheckBox::indicator:unchecked{
    image:url(:/res/chk0.png);
}

QCheckBox::indicator:checked{
    image:url(:/res/chk1.png);
}
</string>
        </property>
        <property name="text">
         <string>Auto uses time as the Y-axis</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="chkMute">
        <property name="text">
         <string>Todo Alert Mute</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="chkDebug">
        <property name="styleSheet">
         <string notr="true">QCheckBox::indicator {
    /* 选择框尺寸 */
    width:22px;
    height:22px;
}</string>
        </property>
        <property name="text">
         <string>Debug Mode</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QToolButton" name="btnBack">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>55</height>
      </size>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
